examples/demo_app/src/App.js

```diff
@@ -24,8 +24,14 @@ const SDBDemo = () => {
 	const username = null;	// set SlashDB username here
 	const apiKey = null;	// set SlashDB API key here
```

Comment:
set `apiKey` here to an empty string with a space in it, ie:
`const apiKey = ' '`

this way the demo app launches properly if the user just runs `npm install` and `npm start`.  if the API key here is `null`, they'll get an exception, same if it's an empty string.

Reply:
shouldn't we fix then app then? let's spend 2 minutes on this at scrum.